src/rustdoc_schema.graphql

```diff
@@ -429,6 +429,29 @@ interface Variant implements Item {
 
   # own edges
   field: [StructField!]
+
+  """
+  The discriminant, if explicitly specified.
+  """
+  discriminant: Discriminant
+}
+
+"""
+https://docs.rs/rustdoc-types/0.28.0/rustdoc_types/struct.Discriminant.html
+https://doc.rust-lang.org/reference/items/enumerations.html
+"""
+type Discriminant {
+  """
+  The expression that produced the discriminant. Preserves the original formatting seen in source code (e.g. suffixes, hexadecimal, and underscores).
+
+  In some cases, when the value is to complex, this may be "`{ _ }`".
```

Comment:
```suggestion
  In some cases, when the value is too complex, this may be a placeholder value such as `"{ _ }"`.
```
We cannot promise a specific placeholder value, because rustdoc doesn't promise that to us. It may at any point decide to switch to `"_"`.

Also note that I swapped the backtick and quote, since I think they were backwards in the original.